packages/wallets/solflare/src/adapter.ts

```diff
@@ -2,21 +2,25 @@ import type { WalletAdapterNetwork, WalletName } from '@solana/wallet-adapter-ba
 import {
     BaseMessageSignerWalletAdapter,
     isIosAndRedirectable,
+    isVersionedTransaction,
     scopePollingDetectionStrategy,
+    SendTransactionOptions,
```

Comment:
```suggestion
    type SendTransactionOptions,
```

Must be imported as types since they are only used as types. Run `pnpm run lint` to check.